lib/engine/game/g_1848/game.rb

```diff
@@ -17,6 +17,8 @@ class Game < Game::Base
 
         BANK_CASH = 10_000
 
+        BOE_START_CASH = 2000
```

Comment:
Nit: Recommend naming BOE_STARTING_CASH to match STARTING_CASH. I'll leave it up to you if you want to follow my recommendation.